apps/vite/src/components/calendar/TimelineView.tsx

```
@@ -0,0 +1,125 @@
+  scope: "month" | "week" | "day"
+}
+
+const currentTime = new Date()
+
+const Day = ({ monthData }: { monthData: NewCalendarData[] }) => {
+  return (
+    <div className="flex">
+      <div className="divide divide-y divide-indigo-200 border-r border-indigo-200">
+        {Array.from({ length: 24 }).map((_, i) => (
+          <div
+            key={i}
```

Biome

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)